pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -10,6 +10,7 @@ import (
 
 	"github.com/aws/aws-sdk-go/aws/awserr"
 
+	"github.com/go-logr/logr"
```

Comment:
Why not use the already imported logger? (line 28)

Reply:
removed